x-pack/plugins/observability_onboarding/public/assets/standalone_agent_setup.sh

```diff
@@ -2,27 +2,121 @@
 
 API_KEY_ENCODED=$1
 API_ENDPOINT=$2
+ELASTIC_AGENT_VERSION=$3
+AUTO_DOWNLOAD_CONFIG=$4
 
 updateStepProgress() {
-  echo "  GET $API_ENDPOINT/step/$1?status=$2"
+  local STEPNAME="$1"
+  local STATUS="$2" # "incomplete" | "complete" | "disabled" | "loading" | "warning" | "danger" | "current"
```

Comment:
At first I was worried about using specific names but as the script and Kibana will always be aligned version wise, we can change it at anytime 👍